tests/unit/sagemaker/huggingface/huggingface_utils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nitpick: replace `Copyright xxxx-xxxx Amazon.com, Inc. or its affiliates. All Rights Reserved.` with `Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.`. See #2532 and #2531.